### PR TITLE
added terraform onboarding tutorial along with few other changes

### PR DESCRIPTION
**Pull request by schoudhury**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by ravilach — DISMISSED** [review]
Looks good to me. -Ravi
Update: Reviewed linter output :-)

File: tutorials/platform/onboard-terraform-provider.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+harness_platform_infrastructure.infra: Creation complete after 3s [id=infrabytf]
+harness_platform_pipeline.pipeline: Creating...
+harness_platform_pipeline.pipeline: Creation complete after 5s [id=pipelinebytf]
+
+Apply complete! Resources: 8 added, 0 changed, 0 destroyed.
+```
+
+## Verify and run pipeline on Harness UI
+
+On Harness UI, you can go to Account Settings --> Organizations to see the new organization. Click View Projects to see the new project. Click the project and go into the "Continuous Delivery" module. When you click Pipelines now, you can see the new pipeline has been created. You can run this pipeline as long as you have previously installed a delegate with name `firstk8sdel` using the Kubernetes delegate instructions from the [Install Delegate](../install-delegate).
```
Comment (ravilach) [inline tutorials/platform/onboard-terraform-provider.md:236]: The link, `"[Install Delegate](../install-delegate)"` might not resolve in MD.  Yarn Linter caught this. 

```
- On source page path = /tutorials/platform/onboard-terraform-provider:

   -> linking to ../install-delegate (resolved as: **/tutorials/install-delegate**)

```

Comment (ravilach) [reply tutorials/platform/onboard-terraform-provider.md:236]: @schoudhury when you get a sec, linter caught that dead link in above comment. Can run the full build linter with the 'yarn build' step locally. 

```
- On source page path = /tutorials/platform/onboard-terraform-provider:

   -> linking to ../install-delegate (resolved as: **/tutorials/install-delegate**)
```

Comment (schoudhury) [reply tutorials/platform/onboard-terraform-provider.md:236]: will fix asap

**Review by ravilach — APPROVED** [review]

**Pull request merged**